tests/integration/testlib/cmd.py

```diff
@@ -20,7 +20,7 @@
 import subprocess
 
 
-def exec_cmd(cmd, env=None):
+def exec_cmd(cmd, env=None, stdin=None):
```

Comment:
How about moving this into a separate commit? It is a prerequisite for the other change but not really related.

Reply:
We introduce it with the usage and it is not that complex. I find it simpler to review this way.
But if you insist, I will comply.

Reply:
In production code, we can "show" how a new unit is used using a test. In tests, when introducing a new "service" option, it is convenient to "show" how to use it or assure it really works by using it (in the same patch).

There are however cases where the  size of the new "service" or test that uses it is too big to make sense, there separation is preferred. Also, when introducing the new "service", only one needs to use it in the same commit, the other (if there are any), better be changed in a separate commit/s.

Again, because it is a grey area, I am fine with either ways, as long as they are merged together (with gerrit, it is harder to assure they get merged together).